tests/mock_cloud_sdk.py

```diff
@@ -0,0 +1,143 @@
+import json
+
+from pasqal_cloud.job import CreateJob, Job
+from pasqal_cloud.device import BaseConfig, EmulatorType
+from pasqal_cloud.batch import Batch
+from pasqal_cloud.utils.responses import PaginatedResponse
+from pasqal_cloud.utils.filters import JobFilters
+
+from uuid import uuid4
+from collections import defaultdict
+
+from unittest.mock import MagicMock
+from typing import Any
+
+
+class MockServer:
```

Comment:
Not necessary for a first version, but would it be feasible to expand the MockServer to be able to inject errors?

Reply:
Yes that's definitely possible!